_providers/daleth.cafe.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Csl. Thomas Lüchow
+status: Teacher and Coach for Adults
```

Comment:
```suggestion
name: daleth.cafe
status: OK
```

This is the name of the instance, not of the person - I'd use daleth.cafe. Also because the name is a double with mehl.cloud otherwise.

The name is what's shown on the start page of https://providers.delta.chat/

That status needs to be `OK`, so we can show in the app that the server is spelled correctly.